content/quickstart/aws/tutorial-ec2-webserver.md

```diff
@@ -51,46 +55,43 @@ In this tutorial, we'll use JavaScript to deploy a simple webserver EC2 instance
 
 1.  To preview and deploy changes, run `pulumi up`. The command shows a preview of the resources that will be created and prompts on whether to proceed with the deployment.  Note that the stack itself is counted as a resource, though it does not correspond to a physical cloud resource.
 
-    ```bash
-    $ pulumi up
-    Previewing update of stack 'webserver-dev'
-    Previewing changes:
-
-        Type                      Name                               Plan       Info
-    +   pulumi:pulumi:Stack       webserver-webserver-dev            create     
-    +   ├─ aws:ec2:SecurityGroup  webserver-secgrp                   create     
-    +   └─ aws:ec2:Instance       webserver-www                      create     
-    
-    info: 3 changes previewed:
-        + 3 resources to create
-    ```
+        $ pulumi up
```

Comment:
Hugo's markdown renderer was having trouble rendering this with the ```bash. To workaround, I can just indent twice (e.g. 8 spaces) and it will be treated as a code snippet and renders as expected. It's not perfect as we lose the language specifier, but for these cases it's not a big deal.